azext_edge/tests/generators.py

```diff
@@ -27,3 +29,36 @@ def generate_random_string(size: int = 36, force_lower: bool = False):
     if not force_lower:
         valid_sequence += string.ascii_uppercase
     return "".join(secrets.choice(valid_sequence) for _ in range(size))
+
+
+def generate_resource_id(
+    resource_group_name: Optional[str] = None,
+    resource_provider: Optional[str] = None,
+    resource_path: Optional[str] = None,
+    resource_subscription: Optional[str] = None
+) -> str:
+    resource_id = f"/subscriptions/{resource_subscription or get_zeroed_subscription()}"
+    if resource_group_name:
+        resource_id += f"/resourceGroups/{resource_group_name}"
+    if resource_provider:
+        resource_id += f"/providers/{resource_provider}"
+    if resource_path:
+        resource_id += resource_path
+    return resource_id
+
+
+def generate_base_endpoint(
+    resource_group_name: Optional[str] = None,
```

Comment:
Was this a copy paste from? https://github.com/Azure/azure-iot-ops-cli-extension/blob/feature/0.7.0/azext_edge/tests/edge/orchestration/resources/conftest.py#L22,

Reply:
oh no you found it - yes. 

will prob move it out after your review

Reply:
will fix this in the next pr